kibble/render/render.go

```diff
@@ -152,6 +175,63 @@ func createLanguage(cfg *models.Config, lang string, locale string) *models.Lang
 		Code:               lang,
 		Locale:             locale,
 		IsDefault:          (lang == cfg.DefaultLanguage),
-		DefinitionFilePath: fmt.Sprintf("%s.all.json", locale),
+		DefinitionFilePath: fmt.Sprintf("%s.all.json", lang),
+	}
+}
+
+func overrideLanguages(site *models.Site, cfg *models.Config) error {
+	if site.Toggles["translations_api"] {
```

Comment:
The toggle is checked in both `overrideLanguages()` and `obtainTranslations()`, I suggest lifting up to wrap the calls to the functions, then each function can reduce it's indentation.

Reply:
Check is now only done on API calls, on file prepare, and write